lib/src/intl_suggestors/intl_migrator.dart

```diff
@@ -163,6 +170,20 @@ class IntlMigrator extends ComponentUsageMigrator {
     // }
   }
 
+  bool isLineIgnored(node) {
+    final ignoreComment = node.findPrevious(node.beginToken).precedingComments;
+    if (ignoreComment != null &&
+        ignoreComment.value().contains(ignoreStatement)) {
+      return true;
+    }
+    return false;
+  }
+
+  bool isFileIgnored(fileContents) {
+    final ignored = fileContents.contains(this.ignoreFile);
+    return ignored;
```

Comment:
Also, I think we'll want to move these outside the specific migrator. There are 4 we use -ConfigsMigrator, IntlMigrator, ConstantStringsMigrator (and intlImporter, but I don't think this applies to that one). So we'll probably want as much as possible to be generic.